docs/proposals/advanced-scheduling.md

```diff
@@ -0,0 +1,106 @@
+# Advanced Scheduling Proposal
+
+Currently, the scheduling of Landscaper deployments to target clusters is based on a simple priority system.
+When scheduling a new Landscaper deployment, the Landscaper Service Controller lists all _visible_ ServiceTargetConfigs (each one representing a target cluster).
+The priority noted of each ServiceTargetConfig is divided by the number of already deployed Landscaper deployments + 1.
+All ServiceTargetConfigs are the sorted in descending order of the computation result. The ServiceTargetConfig with the highest number is selected.
+
+When onboarding large customers this scheduling algorithm is too limited. For large customers the service operator would like to distribute the load of to a defined set of target clusters
+instead of evenly distribute the load evenly across all target clusters.
+
```

Comment:
```suggestion
When onboarding large customers this scheduling algorithm is too limited. For large customers the service operator would like to distribute the load to a defined set of target clusters
instead of evenly distribute the load across all target clusters.

```